test/controllers/static_page_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class StaticPageControllerTest < ActionDispatch::IntegrationTest
+  test "should get home" do
+    get static_page_home_url
+    assert_response :success
+  end
+
+  test "should get help" do
+    get static_page_help_url
+    assert_response :success
+  end
+
+end
```

Comment:
please move folder 'test' to file '.gitignore'. Because we won't work with it, so from now on you don't need to run the command 'a' as shown in the tutorials

Reply:
I will restart the new repo as I just corrupted my file in laptop due to wrong rebase

Reply:
put in .gitignore